packages/sdk/src/actions/simulateDepositTx.ts

```diff
@@ -120,3 +98,31 @@ export async function simulateDepositTx(params: SimulateDepositTxParams) {
 
   return result as unknown as SimulateContractReturnType;
 }
+
+export async function getMaxFillDeadline(
```

Comment:
Do we still need this function? Now that the `fillDeadline` is returned by the API, we'd really prefer integrators to just use that. If we're using it here for setting `fillDeadline` for deposit simulation, I think we could instead sub in a safe value like t+60s, and then override that with the `fillDeadline` returned by the API.

In general the `fillDeadline` is a feature that seems to be unnecessarily modified by others, and it can cause results that are unintuitive for them.